nat-lab/tests/telio.py

```diff
@@ -1049,6 +1051,44 @@ async def get_log(self) -> str:
         await process.execute()
         return process.get_stdout()
 
+    async def get_system_log(self) -> Optional[str]:
+        """
+        Get the system log on the target machine
+        Windows only for now
+        """
+        if self._connection.target_os == TargetOS.Windows:
+            process = self._connection.create_process([
+                "powershell",
+            ])
+            async with process.run():
+                await process.wait_stdin_ready()
+                await asyncio.sleep(1)
```

Comment:
Those sleeps ain't good, can we get some feedback between commands and outputs? Applies for all sleeps in this PR

Reply:
I took this idea from `libtelio/nat-lab/tests/test_fullcone_nat.py`. There are sleeps after `wait_stdin_ready()`. If I understood correctly, this does not mean _really really_ ready, that's why I added those.

Reply:
this maybe, but the following sleeps might be prevented

Reply:
I was worried about that, between issuing a command and fetching stdout, if we'll won't wait for actual output, the log can be corrupted or not retrieved fully.

Reply:
But I'll try without those sleeps ...

Reply:
Tried without additional sleeps so it fails to collect event-logs.

Reply:
you could do something like this:
`"Get-EventLog -LogName Application -Newest 100; echo 'done'\n"`
and then tell output notifier to wait for that `done` output, then you wouldn't need those sleeps

Reply:
same applies for other commands

Reply:
Noice! Changed it!